main.go

```diff
@@ -80,6 +81,7 @@ func main() {
 		setupLog.Error(err, "unable to create k8s clients")
 		os.Exit(1)
 	}
+	registry.Interface = registry.NewRegistry()
```

Comment:
My idea is that https://github.com/openshift-psap/special-resource-operator/pull/165 will remove `registry.Interface` variable, make `NewClusterInfo()` accept `Registry` interface as an argument, store it in the `clusterInfo` and use that object in the `DriverToolkitVersion()`.